documentation/modules/pipelines/pages/pipelines.adoc

```diff
@@ -0,0 +1,8 @@
+= OpenShift Pipelines
+include::master@tekton-tutorial:ROOT:partial$_attributes.adoc[]
+:exprimental:
+:chapter-namespace: pipelines-demos
+:openshift: yes
+:cli: oc
+
+include::master@tekton-tutorial:openshift:partial$_openshift_pipelines.adoc[]
```

Comment:
`Unresolved include directive in modules/pipelines/pages/pipelines.adoc - include::master@tekton-tutorial:openshift:partial$_openshift_pipelines.adoc[]`

Reply:
Thats because of the issue that we discussed over the chat. I resolved it now. The site should be built w/o errors point to my PR and its realted branches in dev-site.yml